ingest/scripts/group_segments.py

```diff
@@ -232,8 +232,13 @@ def main(
 
         row["submissionId"] = joint_key
 
+        # Hash of all metadata fields should be the same if
+        # 1. field is not in keys_to_keep and
+        # 2. field is in keys_to_keep but is ""
```

Comment:
Or None or other falsy values. This is fine if things are always strings, but would be problematic if we sometimes submit integers or booleans and don't coerce to string immediately

Reply:
row is produced from segment_metadata: dict[ str, dict[str, str]] so I'm not sure if this is that risky